src/test/debuginfo/function-prologue-stepping-no-stack-check.rs

```diff
@@ -37,100 +37,79 @@
 // gdb-command:run
 
 // IMMEDIATE ARGS
-// gdb-command:print a
-// gdb-check:$1 = 1
-// gdb-command:print b
-// gdb-check:$2 = true
-// gdb-command:print c
-// gdb-check:$3 = 2.5
+// gdb-command:info args
```

Comment:
Because of #32949, there isn't actually a point for the debugger to break on where the variables for arguments are not in scope, and the first point where it would stop is *before* those variables are actually initialized, so you get garbage locals shadowing the perfectly good args.